src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+ - Automatisch ophogen uitgeschakeld</translation>
+    </message>
+</context>
+<context>
+    <name>ExportDataDialog</name>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="14"/>
+        <source>Export data as CSV</source>
+        <translation>Gegevens exporteren als CSV</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="22"/>
+        <source>Tab&amp;le(s)</source>
+        <translation>&amp;Tabel(-len)</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="57"/>
+        <source>Colu&amp;mn names in first line</source>
+        <translation>&amp;Kolomnamen op eerste regel</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="77"/>
+        <source>Fie&amp;ld separator</source>
+        <translation>&amp;Veldscheidingsteken</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="102"/>
+        <source>,</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="107"/>
+        <source>;</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="112"/>
+        <source>Tab</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="117"/>
+        <source>|</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="122"/>
+        <location filename="../ExportDataDialog.ui" line="192"/>
+        <location filename="../ExportDataDialog.ui" line="254"/>
+        <source>Other</source>
+        <translation>Anders</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="152"/>
+        <source>&amp;Quote character</source>
+        <translation>&amp;Scheidingsteken tekenreeks</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="177"/>
+        <source>&quot;</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="182"/>
+        <source>&apos;</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="222"/>
+        <source>New line characters</source>
+        <translation>Nieuwe-regeltekens</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="244"/>
+        <source>Windows: CR+LF (\r\n)</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="249"/>
+        <source>Unix: LF (\n)</source>
+        <translation></translation>
```

Comment:
I think it's a proper noun that doesn't need translation, so I think you've left the translation blank.
I also did like this in the past, but there was an comments that it is safer to copy the original text for proper nouns
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.